packages/common/components/magazine-issue/item.marko

```diff
@@ -43,14 +43,8 @@ $ const imageLinkAttrs = { title, ...input.imageLinkAttrs };
     </@description>
   </if>
   <@footer-left|{ block }|>
-    <if(input.withFooter)>
-      <cms-link-element class="btn item__button" to=publication path="subscribeUrl" block=block>Subscribe</cms-link-element>
```

Comment:
These links were just intentionally changed this morning. All magazine items will now be missing these links - on all sites. Is that your intention?